timeless/access_control/owner_privileges.py

```diff
@@ -25,12 +25,18 @@ def __location_access(method=None, *args, **kwargs):
 
 def __employee_access(method=None, *args, **kwargs):
     permitted, user = False, flask.g.get("user")
+    if not user:
+        return permitted
+
+    if getattr(user.role, "name", None) == "owner":
```

Comment:
@vladarefiev Here too

Reply:
@paulodamaso I are totally right. I would like to use `role_type` because it is correct way to my mind, but now it's affects a lot of another tests(in another modules), so I have just leave a puzzle for this - https://github.com/timelesslounge/timelessis/pull/497/files#diff-95c37d71b6f5d6c2eedd4fd573f529afR18